Packs/Reco/Integrations/Reco/Reco.py

```diff
@@ -766,6 +812,11 @@ def parse_table_row_to_dict(alert: list[dict[str, Any]]) -> dict[str, Any]:
                 obj[key] = int(obj[key])
             except Exception:
                 demisto.info(f"Could not parse risk level {obj[key]} to int")
+        if key in ["group", "job_title", "departments", "labels"]:
+            try:
+                obj[key] = json.loads(obj[key])
+            except Exception:
+                pass
```

Comment:
Ignoring exceptions is not the best practice. When do you get this exception?
Can I help you figure it out and remove the pass statement?

Reply:
Here it's ok cause we prefer to show some data of the context in compare to empty result